src/erlfmt_scan.erl

```diff
@@ -136,6 +136,8 @@ continue(Scan, Inner0, Loc0, Buffer0) ->
 read_rest(#state{inner = undefined}) ->
     %% reached EOF, no further nodes
     {ok, ""};
+read_rest(#state{scan = _Scan, inner = eof, loc = _Loc, buffer = Buffer}) ->
```

Comment:
When does this happen?

Reply:
This was for no_pragma.erl, which has only one line.
We read a single form to check for pragma, so when we call read_rest, the only line has already been read.